### PR TITLE
Fix API `GET` method for bugzilla comments.

### PR DESCRIPTION
**Pull request by johngian**
* New Bugzilla REST api uses a different endpoint for comments
* Minor lint fixes

### CODE REVIEW

File: remo/remozilla/tasks.py [hunk 1]
```diff
@@ -125,8 +136,9 @@ def fetch_bugs(components=COMPONENTS, days=None):
                         automated_voting_trigger += 1
                     if flag['status'] == '?' and flag['name'] == 'remo-review':
                         bug.pending_mentor_validation = True
-                    if (flag['status'] == '?' and flag['name'] == 'needinfo'
-                            and 'requestee' in flag):
+                    if ((flag['status'] == '?' and
+                         flag['name'] == 'needinfo' and
+                         'requestee' in flag)):
```
Comment (akatsoulas) [inline remo/remozilla/tasks.py]: Nope, it's perfectly happy :)

File: remo/remozilla/tasks.py [hunk 2]
```diff
@@ -29,6 +29,8 @@
        '&product=Mozilla%20Reps&component={component}&'
        'include_fields={fields}&last_change_time={timestamp}&'
        'offset={offset}&limit={limit}')
+COMMENT_URL = ('https://bugzilla.mozilla.org/rest/bug/{id}/comment?'
+               'token={token}')
```
Comment (akatsoulas) [inline remo/remozilla/tasks.py]: You don't need the `token` here in order to get the comments of a bug.

Comment (johngian) [inline remo/remozilla/tasks.py]: You need this, these specific bugs are not publicly accessible.

Comment (akatsoulas) [inline remo/remozilla/tasks.py]: You do need this for non public bugs. Ignore the previous comment.

**Comment by akatsoulas** [comment]
r+wc :cocktail:

**Pull request merged**